README.md

```diff
@@ -1,5 +1,18 @@
-# license-list-XML
-This is a workspace for developing and implementing a new XML source format for the SPDX License List. This source format is (or will be) used to generate the authoritative file formats for licenses on the SPDX License List. These authoritative [target/destination formats](https://github.com/spdx/license-list-data) will remain the same for the time being.
+# SPDX License List Source Files
```

Comment:
nit: I think we can drop "Files" to shorten this header without dropping information.

Reply:
agree